Stresser/main.cpp

```diff
@@ -79,20 +86,28 @@ int wmain(int argc, PWCHAR argv[])
 
 		// Create background thread to fetch endpoint configuration from the server:
 		// TODO: Move from here !
-		StandardThread communication([&endpointController, &policyController, &endpoint, &ruleController]()
+		StandardThread communication([&endpointController, &policyController, &endpointId, &ruleController, &artifactsVector]()
 			{
-				EndpointEntity endpointEntity = endpointController.getEndpoint(endpoint.GetID());
+				EndpointEntity endpointEntity = endpointController.getEndpoint(endpointId);
 				DEBUG_PRINT(endpointEntity);
 
 				// Check if there is CTRL + C signal:
-				while (WAIT_TIMEOUT == WaitForSingleObject(g_shutdownEvent.get(), 1000))
+				while (WAIT_TIMEOUT == WaitForSingleObject(g_shutdownEvent.get(), 10000))
```

Comment:
CHORE for both of us: 
Move this constant (wait time) to StresserApplication object.
The same goes for other constants